tests/installation/bootloader_svirt.pm

```diff
@@ -120,16 +119,16 @@ sub run() {
     $svirt->add_vnc({port => '5901'});
 
     if ($vmm_family eq 'kvm') {
-        $svirt->add_interface({type => 'network', source => {network => 'default'}, model => {type => 'virtio'}});
+        $svirt->add_interface({type => 'bridge', source => {bridge => 'br0'}, model => {type => 'virtio'}});
     }
     elsif ($vmm_family eq 'xen') {
         if ($vmm_type eq 'hvm') {
-            $svirt->add_interface({type => 'network', source => {network => 'default'}, model => {type => 'netfront'}});
+            $svirt->add_interface({type => 'bridge', source => {bridge => 'br0'}, model => {type => 'netfront'}});
             # emulator is not being set for Xen HVM automatically
             $svirt->add_emulator({emulator => '/usr/lib/xen/bin/qemu-system-i386'});
         }
         elsif ($vmm_type eq 'linux') {
-            $svirt->add_interface({type => 'network', source => {network => 'default'}});
+            $svirt->add_interface({type => 'bridge', source => {bridge => 'br0'}});
         }
```

Comment:
Every available server I have uses bridge instead of `network='default'`. Not sure how's that on our infrastructure but without this change merged I need to keep it with me when testing new code.